draft-ietf-quic-multipath.md

```diff
@@ -310,6 +310,18 @@ Each endpoint associates a Receiver Packet Number space to each Path Identifier
 that it provides to the peer. Each endpoint associates a Sender Packet Number space 
 to each Path Identifier received from the peer.
 
+Endpoints use the same Path ID for one specific path in both directions. 
+For a client-initiated path, the client decides which Path ID used for the new path, 
+it picks one of the server allocated CID with the specified Path ID. 
+Then the client send a PATH_CHALLENGE with the chosen CID. If the server receives the PATH_CHALLENGE, 
+it picks a Connection ID with the same path ID for sending the PATH_RESPONSE.
+
+The client MUST choose a previously unused Path ID for which both endpoints have already issued at least one connection ID.
+If the server receives a PATH_CHALLENGE before receiving MP_NEW_CONNECTION_ID 
+for the specify path, it MAY choose to ignore the PATH_CHALLENGE, or it can
+choose to send PATH_RESPONSE until it receives the MP_NEW_CONNECTION_ID containing
+the corresponding Path ID arrives.
```

Comment:
It is not clear to me how the server should send the PATH_RESPONSE. Over which Path ID?

Reply:
The server shall pick a CID provided by the client with the same path_ID as the server provided CID that was used by the client to send the PATH_CHALLENGE.

Reply:
It's written in the last sentence of the previous paragraph

Reply:
Oh, right. I thought it was possible to send the PATH_RESPONSE when you do not have the associated path ID (hence my misunderstanding), but by rereading, this is fine. Maybe rephrasing it to make it a bit clearer?
```suggestion
choose to send the PATH_RESPONSE frame upon reception of a
MP_NEW_CONNECTION_ID frame containing the corresponding Path ID.
```